protostar.py

```diff
@@ -13,7 +15,16 @@ def regexp(input_string: str) -> Pattern:
 
 def directory(directory_path: str) -> Path:
     pth = Path(directory_path)
-    assert pth.is_dir(), "Given argument is not a valid directory path"
+    assert pth.is_dir(), f'"{str(pth)}" is not a valid directory path'
+    return pth
+
+
+def cairo_file(file_path: str) -> Path:
```

Comment:
That's just the type we provide for `argparse`, and thus it's named `cairo_file` since we provide it as a `type` parameter. I think if we have another case of validation of cairo files extension, we can consider extracting it somewhere else